tox.ini

```diff
@@ -16,7 +16,7 @@ commands =
          print(settings.FLOW_EXECUTOR[\"CONTAINER_IMAGE\"])" \
     )\'
 # run tests
-    coverage run --source=resolwe_bio tests/manage.py test --noinput -v 2 {env:TEST_SUITE:resolwe_bio}
+    coverage run tests/manage.py test {env:TEST_SUITE:resolwe_bio} --noinput --verbosity=2
```

Comment:
@mstajdohar, did you intentionally omit `--parellel` in this command?
If yes, don't forget to also include `coverage combine` as the next command as [in Resolwe](https://github.com/genialis/resolwe/blob/987328440128926ee4daefa0b0d55e6af35b398d/tox.ini#L13-L15).